translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\n"
+"\t\t# Delete all pods\n"
+"\t\tkubectl delete pods --all"
+msgstr ""
+"\n"
+"\t\t# Remove um pod usando o tipo e nome especificado no pod.json.\n"
+"\t\tkubectl delete -f ./pod.json\n"
+"\n"
+"\t\t# Remove um pod baseado no tipo e nome no JSON passado na entrada de "
+"comando(stdin).\n"
+"\t\tcat pod.json | kubectl delete -f -\n"
+"\n"
+"\t\t# Remove pods e serviços com os nomes “baz” and “foo”\n"
+"\t\tkubectl delete pod,service baz foo\n"
+"\n"
+"\t\t# Remove pods e serviços com label name=myLabel.\n"
+"\t\tkubectl delete pods,services -l name=myLabel\n"
+"\n"
+"\t\t# Remove um pod com um mínimo de delay\n"
+"\t\tkubectl delete pod foo —now\n"
+"\n"
+"\t\t# Força a remoção de  um pod em um node morto\n"
+"\t\tkubectl delete pod foo —grace-period=0 —force\n"
+"\n"
+"\t\t# Remove todos os pods\n"
+"\t\tkubectl delete pods —all"
+
+#: pkg/kubectl/cmd/describe.go:54
+msgid ""
+"\n"
+"\t\t# Describe a node\n"
+"\t\tkubectl describe nodes kubernetes-node-emt8.c.myproject.internal\n"
+"\n"
+"\t\t# Describe a pod\n"
+"\t\tkubectl describe pods/nginx\n"
+"\n"
+"\t\t# Describe a pod identified by type and name in \"pod.json\"\n"
+"\t\tkubectl describe -f pod.json\n"
+"\n"
+"\t\t# Describe all pods\n"
+"\t\tkubectl describe pods\n"
+"\n"
+"\t\t# Describe pods by label name=myLabel\n"
+"\t\tkubectl describe po -l name=myLabel\n"
+"\n"
+"\t\t# Describe all pods managed by the 'frontend' replication controller (rc-"
+"created pods\n"
+"\t\t# get the name of the rc as a prefix in the pod the name).\n"
+"\t\tkubectl describe pods frontend"
+msgstr ""
+"\n"
+"\t\t# Descreve um node\n"
+"\t\tkubectl describe nodes kubernetes-node-emt8.c.myproject.internal\n"
+"\n"
+"\t\t# Descreve um pod\n"
+"\t\tkubectl describe pods/nginx\n"
+"\n"
+"\t\t# Descreve um pod identificado pelo tipo e com nome no “pod.json”\n"
+"\t\tkubectl describe -f pod.json\n"
+"\n"
+"\t\t# Descreve todos os pods\n"
+"\t\tkubectl describe pods\n"
+"\n"
+"\t\t# Descreve os pods com label name=myLabel\n"
+"\t\tkubectl describe po -l name=myLabel\n"
+"\n"
+"\t\t# Descreve todos os pods gerenciado por ‘frontend’ replication "
+"controller (rc-created pods\n"
+"\t\t# tem o nome de rc como prefixo no nome do pod).\n"
+"\t\tkubectl describe pods frontend"
+
+#: pkg/kubectl/cmd/drain.go:165
+msgid ""
+"\n"
+"\t\t# Drain node \"foo\", even if there are pods not managed by a "
+"ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet on it.\n"
+"\t\t$ kubectl drain foo --force\n"
+"\n"
+"\t\t# As above, but abort if there are pods not managed by a "
+"ReplicationController, ReplicaSet, Job, DaemonSet or StatefulSet, and use a "
+"grace period of 15 minutes.\n"
+"\t\t$ kubectl drain foo --grace-period=900"
+msgstr ""
+"\n"
+"\t\t# Drena o node “foo”, mesmo se os pods não são gerenciados por um "
+"ReplicationController, ReplicaSet, Job, DaemonSet ou StatefulSet.\n"
+"\t\t$ kubectl drain foo —force\n"
+"\n"
+"\t\t# Mesmo que acima, mas é abortado se os pods não são gerenciados por um "
```

Comment:
Here is my strictly personal opinion. The literal translation of "aborted" is pretty strong in portuguese, so I would use "interrompido" instead of "abortado"